test/internet.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { describe, expect, it, vi } from 'vitest';
 import { faker } from '../lib';
+import validator from 'validator';
 
 describe('internet.js', () => {
```

Comment:
Oh I missed that one

```suggestion
describe('internet', () => {
```